src/goto-programs/show_goto_functions_json.cpp

```diff
@@ -0,0 +1,106 @@
+/*******************************************************************\
+
+Module: Goto Program
+
+Author: Thomas Kiley
+
+\*******************************************************************/
+
+#include <goto-programs/show_goto_functions_json.h>
```

Comment:
By convention, this should be the last include file, and it should just be `#include "show_goto_functions_json.h"`